packages/headless/src/features/result-preview/result-preview-actions.ts

```diff
@@ -35,3 +38,13 @@ export const fetchResultContent = createAsyncThunk<
     };
   }
 );
+
+/**
+ * Updates the maximum preview size allowed for rendering in quickview.
+ * @param maximumPreviewSize (number) The new maximum preview size allowed for rendering in quick view.
+ */
+export const updateMaximumPreviewSize = createAction(
+  'quickview/updateMaximumPreviewSize',
+  (payload: number) =>
+    validatePayload(payload, new NumberValue({required: true, min: 0}))
+);
```

Comment:
I created this function but have not used/called it anywhere. Any thoughts?

Reply:
I don't think we need to be able to update the maximum preview size since it can be done at a controller level.

Reply:
Yes that makes sense. Should I then just remove it if it serves no purpose?

Reply:
yep, let's remove this action